### ISSUE TITLE
kolodaSpeedThatCardShouldDrag should be optional and use the default value when not provided

### ISSUE THREAD

**Issue by benpackard** [Yalantis/Koloda#314]
Having recently updated to version `4.1` I find that I now need to implement `kolodaSpeedThatCardShouldDrag`. This should not be necessary for uses where the default speed is sufficient.

This could be fixed with a default implementation that returns `DragSpeed.default`, e.g.:

```
public extension KolodaViewDataSource {
    func kolodaSpeedThatCardShouldDrag(_ koloda: KolodaView) -> DragSpeed {
	    return .default
    }
}
```

If this has support, I can send a pull request.

**Comment by kaamoss** [comment]
To that end, at least until this is changed, maybe someone could update the README docs to reflect that KolodaViewDataSource requires defining this func?

**Comment by freaqed** [comment]
Hi, @kaamoss!
Please take a look at the latest version. Now it's fixed to use delegates value if it's present

**Comment by kaamoss** [comment]
@freaqed Awesome, thank you! I'm sure the next developer who comes along will greatly appreciate it.

**Comment by benpackard** [comment]
I don't think is is fixed - I am using `4.3` and I still have to implement `kolodaSpeedThatCardShouldDrag` in my datasource, which shouldn't be necessary if I want to use the default value. My proposal is to simply add a default implementation to `KolodaViewDataSource`, as is the case with `viewForCardOverlayAt` which returns `nil` if not implemented.

**Comment by benpackard** [comment]
I submitted a pull request: https://github.com/Yalantis/Koloda/pull/325
